### ISSUE TITLE
self-hosted runner issues

### ISSUE THREAD

**Issue by rsliotta** [anton-yurchenko/git-release#43]
Hello...

Two different issues....  

The workspace seems to be hardcoded for github and the self_hosted_runners use a different path.  I tried to put and absolute patch and that did not work
```
Run docker://antonyurchenko/git-release:latest
  with:
    args: dist/artifacts/*
  
  env:
    GITHUB_TOKEN: ***
    CHANGELOG_FILE: /k3os_builder/CHANGES.md
    DRAFT_RELEASE: false
    PRE_RELEASE: 
    ALLOW_EMPTY_CHANGELOG: false
    ALLOW_TAG_PREFIX: true
/usr/bin/docker run --name antonyurchenkogitreleaselatest_efc5ea --label fc8a87 --workdir /github/workspace --rm -e GITHUB_TOKEN -e CHANGELOG_FILE -e DRAFT_RELEASE -e PRE_RELEASE -e ALLOW_EMPTY_CHANGELOG -e ALLOW_TAG_PREFIX -e INPUT_ARGS -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/liottar/actions-runner/_work/_temp/_github_home":"/github/home" -v "/home/liottar/actions-runner/_work/_temp/_github_workflow":"/github/workflow" -v "/home/liottar/actions-runner/_work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/liottar/actions-runner/_work/k3os_builder/k3os_builder":"/github/workspace" antonyurchenko/git-release:latest dist/artifacts/*
INFO 'git-release' version: 3.4.4                 
WARNING 'PRE_RELEASE' is not equal to 'true', assuming 'false' 
WARNING 'ALLOW_TAG_PREFIX' enabled                   
FATAL changelog file '/github/workspace//k3os_builder/CHANGES.md' not found! 
```

I am also building for ARM64 and have this...

```
Run docker://antonyurchenko/git-release:latest
/usr/bin/docker run --name antonyurchenkogitreleaselatest_4ff6bf --label fc8a87 --workdir /github/workspace --rm -e GITHUB_TOKEN -e CHANGELOG_FILE -e DRAFT_RELEASE -e PRE_RELEASE -e ALLOW_EMPTY_CHANGELOG -e ALLOW_TAG_PREFIX -e INPUT_ARGS -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/liottar/actions-runner/_work/_temp/_github_home":"/github/home" -v "/home/liottar/actions-runner/_work/_temp/_github_workflow":"/github/workflow" -v "/home/liottar/actions-runner/_work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/liottar/actions-runner/_work/k3os_builder/k3os_builder":"/github/workspace" antonyurchenko/git-release:latest dist/artifacts/*
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
standard_init_linux.go:219: exec user process caused: exec format error
```

**Comment by anton-yurchenko** [comment]
Hello @rsliotta,

Currently, ARM64 is not supported, but I will surely look into adding this.

Regarding the self-hosted runner, **git-release** relies on `GITHUB_WORKSPACE` environmental variable in order to resolve the workspace directory. Changelog file, as the rest of the artifacts are expected to be in the workspace directory.

Please provide more information:

- We will have to look at the environmental variables that start with `GITHUB_*`
- Provide a path to the current working directory
- We will have to validate that `CHANGELOG.md` is indeed in the workspace

Try printing this out with a step similar to:

```yaml
      - name: Debug
        run: |
          echo "---> Environmental Variables:"
          printenv | grep "GITHUB_"
          echo "---> Current Directory:"
          pwd
          echo "---> Changelog File:"
          ls -l CHANGELOG.md
```

**Comment by anton-yurchenko** [comment]
Meanwhile, ARM64 support has been released as a part of v3.5.0 🎉

**Comment by anton-yurchenko** [comment]
Closing for inactivity
